src/js/pinny.js

```diff
@@ -195,12 +203,22 @@
         },
 
         _bindEvents: function() {
+            var container = this.$pinny;
+
             // Block scrolling on anything but pinny content
-            this.$pinny.on('touchmove', function(e) {
+            container.on('touchmove', function(e) {
                 if (!$(e.target).parents().hasClass(classes.CONTENT)) {
                     e.preventDefault();
                 }
             });
+
+            if ($.os.ios && $.os.major === 7) {
```

Comment:
I'm really confused here. I thought we were doing synthetic resize if this was iOS 7. Is this not the case?